lib/Layout/Resolver/Form/TargetType/Mapper/SingleTaxon.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Layouts\Sylius\Layout\Resolver\Form\TargetType\Mapper;
+
+use Netgen\ContentBrowser\Form\Type\ContentBrowserType;
+use Netgen\Layouts\Layout\Resolver\Form\TargetType\Mapper;
+
+/**
+ * @deprecated this class will be renamed to simply Taxon in next major release
```

Comment:
Please use concrete version: `in 2.0`

Reply:
@emodric Fixed in https://github.com/netgen-layouts/layouts-sylius/pull/17/commits/057d41669cd3128b87eaf8063b07be5151bc5382